tailcfg/tailcfg.go

```diff
@@ -2209,6 +2213,9 @@ const (
 
 	// NodeAttrsTailFSAccess enables accessing shares via TailFS.
 	NodeAttrsTailFSAccess NodeCapability = "tailfs:access"
+
+	// NodeAttrsExitNodeSuggestionCandidate labels a node as an exit node suggestion candidate.
+	NodeAttrsExitNodeSuggestionCandidate NodeCapability = "exit-node-suggestion-candidate"
```

Comment:
That seems a little overly verbose and tentative. `NodeAttrsSuggestedExitNode` / `suggested-exit-node` is probably plenty.